godot-kotlin/godot-library/build.gradle.kts

```diff
@@ -10,19 +10,20 @@ kotlin {
     linuxX64("linux")
     mingwX64("windows")
 
-    val internalSourceSet = sourceSets.create("nativeInternal")
-    val coreSourceSet = sourceSets.create("nativeCore") { dependsOn(internalSourceSet) }
-    val generatedSourceSet = sourceSets.create("nativeGen") { dependsOn(coreSourceSet) }
-    val publicSourceSet = sourceSets.create("nativePublic") { dependsOn(generatedSourceSet) }
+    // val internalSourceSet = sourceSets.create("nativeInternal")
```

Comment:
so that cinterop will work (this will conflict with what @piiertho is working on)